src/common/SurgeStorage.cpp

```diff
@@ -380,6 +381,7 @@ SurgeStorage::SurgeStorage(const SurgeStorage::SurgeStorageConfig &config) : oth
     userFXPath = userDataPath / "FX Presets";
     userMidiMappingsPath = userDataPath / "MIDI Mappings";
     userModulatorSettingsPath = userDataPath / "Modulator Presets";
+    userWavetableScriptPath = userDataPath / "Wavetable Script Presets";
```

Comment:
As discussed we will want to merge .wtscripts with normal wavetables, so separate path shouldn't be needed. Can resolve in a subsequent PR, although it chould really be as simple as adding a new supported extension to `SurgeStorage::refresh_wtlistFrom()`.